server/controllers/articleController.js

```diff
@@ -361,6 +361,40 @@ class ArticleController {
     const categories = await Categories.findAll();
     return successStat(res, 200, 'categories', categories);
   }
+
+  /**
+   * @description sharing an article on facebook
+   * @param {Object} req - request object
+   * @param {Object} res - response object
+   * @returns {String} returns a link to the article on facebook website
+  */
+  static async shareOnFacebook(req, res) {
+    const article = await models.Article.findOne({
+      where: { slug: req.params.slug }
+    });
+    if (!article) {
+      return errorStat(res, 404, 'not found');
+    }
+    const facebookSDK = `https://www.facebook.com/sharer/sharer.php?u=${article.slug}`;
+    return successStat(res, 200, 'share link', `${facebookSDK}`);
```

Comment:
Nice job @chiboycalix. It's amazing how fast you guys are able to research and come up with your tasks. Good job. However, I think it doesn't suffice to send the url in the response body. I'm of the opinion that you do something like res.redirect(`${facebookSDK}`) so the user is redirected to the facebook page to share his resource once he hits the endpoint.

Reply:
Is there a story to count the amount of shares? If not, I think you need to factor that too. Nonetheless, nice work bro.

Reply:
Thanks for your comments. I'll do just that.